dipy/direction/peaks.py

```diff
@@ -157,8 +157,64 @@ def peak_directions(odf, sphere, relative_peak_threshold=.5,
     return directions, values, indices
 
 
+def _pam_from_attrs(sphere, peak_indices, peak_values, peak_dirs,
+                    gfa, qa, shm_coeff, B, odf):
+    """
+    Construct a PeaksAndMetrics object from its attributes.
+
+    This is also useful for pickling/unpickling of these object (see also
+    :func:`__reduce__` below).
+
+    Parameters
+    ----------
+    sphere : `Sphere` class instance.
+        Sphere for discretization.
+    peak_indices : ndarray
+        Indices (in sphere vertices) of the peaks in each voxel.
+    peak_values : ndarray
+        The value of the peaks.
+    peak_dirs : ndarray
+        The direction of each peak.
+    gfa : ndarray
+        The Generalized Fractional Anisotropy in each voxel.
+    qa : ndarray
+        Quantitative Anisotropy in each voxel.
+    shm_coeff : ndarray
+        The coefficients of the spherical harmonic basis for the ODF in
+        each voxel.
+    B : ndarray
+        The spherical harmonic matrix, for multiplication with the
+        coefficients.
+    odf : ndarray
+        The orientation distribution function on the sphere in each voxel.
+
+    Returns
+    -------
+    PeaksAndMetrics class instance.
```

Comment:
I think docstring standard says something like:

```
Returns
-------
pam : PeaksAndMetrics class instance
```

or similar (give name to output arg).